src/probnum/problems/zoo/linalg/_suitesparse_matrix.py

```diff
@@ -56,9 +56,11 @@ def suitesparse_matrix(
         import requests  # pylint: disable=import-outside-toplevel
     except ImportError as err:
         raise ImportError(
-            "Cannot query SuiteSparse Matrix collection without optional dependency ",
-            "`requests`. Install ProbNum with optional dependencies for the problem zoo via",
-            " `pip install probnum[zoo]` or install requests directly: `pip install requests`.",
+            "Cannot query SuiteSparse Matrix collection without",
+            "optional dependency `requests`. Install ProbNum with",
+            "optional dependencies for the problem zoo via",
+            "`pip install probnum[zoo]` or install requests",
+            "directly: `pip install requests`.",
```

Comment:
Whitespaces are missing behind 'without', 'with', 'via', and 'requests'. Without the extra whitespaces, the error message will be displayed incorrectly. 

```suggestion
            "Cannot query SuiteSparse Matrix collection without ",
            "optional dependency `requests`. Install ProbNum with ",
            "optional dependencies for the problem zoo via ",
            "`pip install probnum[zoo]` or install requests ",
            "directly: `pip install requests`.",
```

Reply:
Hm, I initially added whitespaces after those words, however I did my own testing with `print()` and noticed that whitespaces were added where necessary as seen in the attached image. I might be just misunderstanding how Python string manipulation work though.. 

![image](https://user-images.githubusercontent.com/66120338/164311942-2d49c1cf-cbfc-4cad-912a-1e8fc2ae1078.png)

Reply:
Thanks for clarifying this. I did not know about the whitespace-print thing!

Technically, however, this is an exception not a print. For exceptions, this return-style produces odd outputs. Note how the "a", "b", "c" below is not concatenated:

![Screenshot from 2022-04-23 09-56-50](https://user-images.githubusercontent.com/45484438/164885868-bd7b0f02-1a0f-4a8a-9849-dd39e9d96e94.png)

However, this seems to be a little out of scope for this PR, which should only be about line-too-long statements. The linebreak issues I am talking about have been there before, so it is not your responsibility to fix them :) Let's handle this in a different pull request.

Reply:
Ah gotcha, I guess string concatenations are handled somewhat different for Exceptions in Python. Also, can I clarify whether the commas between the strings are meant to be displayed as well? I just followed the original code as I assumed that was the expected behaviour however, after seeing your image, it struck me as odd. Perhaps the following was the intended outcome? 
![image](https://user-images.githubusercontent.com/66120338/164887670-53837693-d589-4313-9916-fe553ea93e91.png)

Reply:
yes, something like this would be desirable in my opinion